src/lib/flight_tasks/tasks/Offboard/FlightTaskOffboard.cpp

```diff
@@ -153,7 +155,7 @@ bool FlightTaskOffboard::update()
 		_position_setpoint.setNaN(); // Don't require any position/velocity setpoints
 		_velocity_setpoint.setNaN();
 		_acceleration_setpoint = Vector3f(0.f, 0.f, 100.f); // High downwards acceleration to make sure there's no thrust
-		return true;
+		return ret;
```

Comment:
omg we should to get rid of these early returns, not here.

Reply:
Yes, it's horrible :|